### PR TITLE
Add info about switching versions of BBDO

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Add info about switching versions of BBDO

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 1]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+   - On the central server and on remote servers:
+
+     ```shell
+     stop cbd centengine
+     ```
+
+   - On the pollers:
+
+     ```shell
+     stop cbd centengine
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:355]:
```suggestion
     systemctl stop centengine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 2]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+
+1. On the central server, go to **Configuration > Pollers > Broker configuration**.
+2. Select the server you want, and on the **General** tab, in section **Advanced options**, select the version of BBDO you want from the **BBDO version** list. Then click on **Save**.
+3. Do the same with all the elements listed on page **Configuration > Pollers > Broker configuration**.
+4. [Deploy the configuration](../monitoring/monitoring-servers/deploying-a-configuration.md) for all servers.
+5. Stop the following services:
+   - On the central server and on remote servers:
+
+     ```shell
+     stop cbd centengine
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:349]:
```suggestion
     systemctl stop cbd centengine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md (hunk 1 above)
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:355]: pas de cbd sur un poller

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 3]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+     ```
+
+6. Start the following services:
+   - On the central server and on remote servers:
+
+     ```shell
+     start cbd centengine
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:362]:
```suggestion
     systemctl start cbd centengine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 4]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+     ```
+
+   - On the pollers:
+
+     ```shell
+     start cbd centengine
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:368]: same pas de cbd sur les pollers

Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:368]:
```suggestion
     systemctl start centengine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 5]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+- remote broker:
+
+  ```shell
+  /var/log/centreon-broker/<remote_name>-broker-master.log
+  ```
+
+- poller module:
+
+  ```shell
+  /var/log/centreon-broker/<poller_name>-module.log
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:388]: c'est le log file du poller sur le central

File: versioned_docs/version-22.04/developer/developer-broker-bbdo.md [hunk 6]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+- remote broker:
+
+  ```shell
+  /var/log/centreon-broker/<remote_name>-broker-master.log
+  ```
+
+- poller module:
+
+  ```shell
+  /var/log/centreon-broker/<poller_name>-module.log
```
Comment (chgautier) [inline versioned_docs/version-22.04/developer/developer-broker-bbdo.md:388]: on the central server as stated above

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 7]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+- central broker:
+
+  ```shell
+  /var/log/centreon-broker/central-broker-master.log
+  ```
+
+- remote broker:
+
+  ```shell
+  /var/log/centreon-broker/<remote_name>-broker-master.log
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:382]:
```suggestion
tail  /var/log/centreon-broker/<remote_name>-broker-master.log
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 8]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+     ```shell
+     start cbd centengine
+     ```
+
+You can check in the logs which version of BBDO is active for a server:
+
+- central broker:
+
+  ```shell
+  /var/log/centreon-broker/central-broker-master.log
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:376]:
```suggestion
tail  /var/log/centreon-broker/central-broker-master.log
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md (hunk 5 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:388]:
```suggestion
tail  /var/log/centreon-broker/<poller_name>-module.log
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md (hunk 1 above)
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:355]:
```suggestion
     stop centengine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md (hunk 4 above)
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:368]:
```suggestion
     start centengine
```

File: versioned_docs/version-22.04/developer/developer-broker-bbdo.md [hunk 9]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+   - On the central server and on remote servers:
+
+     ```shell
+     stop cbd centengine
+     ```
+
+   - On the pollers:
+
+     ```shell
+     stop cbd centengine
```
Comment (chgautier) [inline versioned_docs/version-22.04/developer/developer-broker-bbdo.md:355]:
```suggestion
     stop centengine
```

File: versioned_docs/version-22.04/developer/developer-broker-bbdo.md [hunk 10]
```diff
@@ -331,3 +331,65 @@ by the client.
...
+   - On the central server and on remote servers:
+
+     ```shell
+     start cbd centengine
+     ```
+
+   - On the pollers:
+
+     ```shell
+     start cbd centengine
```
Comment (chgautier) [inline versioned_docs/version-22.04/developer/developer-broker-bbdo.md:368]:
```suggestion
     start centengine
```

**Review by chgautier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 11]
```diff
@@ -331,3 +331,71 @@ by the client.
...
+     systemctl start centengine
+     ```
+
+You can check in the logs which version of BBDO is active for a server:
+
+- central broker:
+
+  ```shell
+  tail /var/log/centreon-broker/central-broker-master.log
+  ```
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:383]:
```suggestion
- central broker:

  ```shell
  tail /var/log/centreon-broker/central-{broker,rrd,module}-master.log
  ```
```

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:383]: There are three central broker log files

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md [hunk 12]
```diff
@@ -331,3 +331,71 @@ by the client.
...
+
+- remote broker:
+
+  ```shell
+  tail /var/log/centreon-broker/<remote_name>-broker-master.log
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:388]:
```suggestion
  tail /var/log/centreon-broker/<remote_name>-{broker,rrd,module}-master.log
```

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.04/developer/developer-broker-bbdo.md:388]: Same as for central

**Review by chgautier — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**